HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController+Helpers.swift

```diff
@@ -0,0 +1,806 @@
+                                                                   poweredByGiniStrings: stringsProvider.poweredByGiniStrings,
+                                                                   moreInformationConfiguration: configurationProvider.moreInformationConfiguration,
+                                                                   moreInformationStrings: stringsProvider.moreInformationStrings)
+        paymentProvidersBottomViewModel.viewDelegate = self
+        paymentProvidersBottomViewModel.documentId = documentId
+        return BanksBottomView(viewModel: paymentProvidersBottomViewModel, bottomSheetConfiguration: configurationProvider.bottomSheetConfiguration)
+    }
+    
+    /**
+     Loads the payment review screen for the specified document or for the provided payment information
+
+     This method fetches data for review based on the provided document ID. If the configuration
+     allows for invoice handling without a document, it directly loads the payment review screen
+     using the provided payment information provided.
+
+     - Parameters:
+       - documentId: An optional identifier for the document being reviewed.
+       - paymentInfo: An optional `PaymentInfo` object containing details about the payment.
+       - trackingDelegate: An optional delegate for tracking events related to Gini Health.
+       - completion: A closure that is called with the resulting `UIViewController` and an optional
+         `GiniHealthError` once the loading process is complete.
+     */
+    func loadPaymentReviewScreenFor(trackingDelegate: GiniHealthTrackingDelegate?, completion: @escaping (UIViewController?, GiniHealthError?) -> Void) {
+        previousPresentedView = nil
+        if !GiniHealthConfiguration.shared.useInvoiceWithoutDocument {
+            guard let documentId else {
+                completion(nil, nil)
+                return
+            }
+            self.isLoading = true
+            self.giniSDK.fetchDataForReview(documentId: documentId) { [weak self] result in
+                self?.isLoading = false
+                switch result {
+                    case .success(let data):
+                        guard let self else {
+                            completion(nil, nil)
+                            return
+                        }
+                        self.preparePaymentReviewViewController(data: data, paymentInfo: nil, completion: completion)
+                    case .failure(let error):
+                        completion(nil, error)
+                }
+            }
+        } else {
+            loadPaymentReviewScreenWithoutDocument(paymentInfo: paymentInfo, trackingDelegate: trackingDelegate, completion: completion)
+        }
+    }
+    
+    private func loadPaymentReviewScreenWithoutDocument(paymentInfo: GiniInternalPaymentSDK.PaymentInfo?, trackingDelegate: GiniHealthTrackingDelegate?, completion: @escaping (UIViewController?, GiniHealthError?) -> Void) {
```

Comment:
[request] The method signature seems too long. Can we move each parameter on a new line?